components/quality-dashboard/frontend/route.yaml

```diff
@@ -0,0 +1,13 @@
+kind: Route
+apiVersion: route.openshift.io/v1
+metadata:
+  name: quality-frontend-route
+  namespace: quality-dashboard
+spec:
```

Comment:
Would it be possible to use edge termination to have secured route (https)?

Reply:
I would suggest to merge this PR, fix it in quality-dashboard and then create new PR with fix? WDYT?
I have tried something like
```
tls:
    termination: edge
```
but when I have added this, the frontend did not connect to backend for some reason.